### ISSUE TITLE
Input border in firefox

### ISSUE THREAD

**Issue by cabutler10** [zepdev/zeppelin-element-library#112]
![image](https://user-images.githubusercontent.com/4421077/62711512-75735a00-b9f9-11e9-8771-61102ce82f8b.png)

**Comment by byjs-dev** [comment]
This should be the css outline property.

**Comment by cabutler10** [comment]
@byjs-dev good point. Let's wait on this until we get new designs and maybe we can fix it by removing some work to accommodate ie9?
